src/transformers/pipelines.py

```diff
@@ -636,7 +636,7 @@ def __call__(self, *texts, **kwargs):
                 if self.model.config.id2label[label_idx] not in self.ignore_labels:
                     answer += [
                         {
-                            "word": self.tokenizer.decode([int(input_ids[idx])]),
+                            "word": self.tokenizer.convert_ids_to_tokens(int(input_ids[idx])),
```

Comment:
Is there a reason you changed from `decode` to `convert_ids_to_tokens`?

Reply:
Yes, I detailed it in the merge request :) 

In short, decode(...) tries to revert multiple tokens to the original word, but since tokens are iterated over on-by-one this makes no sense (or maybe I am missing something). Or rather, it does nothing for BERT, but it removes the leading '_' in Albert tokens which makes it impossible to reconstruct words from tokens since that conversion relies on the leading '_'.

Reply:
Oops, indeed, I missed this! I'm asking @mfuntowicz who worked on the pipelines to verify :)

Reply:
Yap I agree on the behaviour you describe. It might change in the near future, as we were discussing to output the whole word NER class and not on a subtoken basis (as currently done). 

I'm fine with the proposed change.